counterpartylib/lib/micropayments/control.py

```diff
@@ -0,0 +1,729 @@
+# coding: utf-8
+# Copyright (c) 2016 Fabian Barkhau <fabian.barkhau@gmail.com>
+# License: MIT (see LICENSE file)
+
+
+import copy
+import pycoin
+from . import validate
+from . import exceptions
+from micropayment_core import util
```

Comment:
is this file part of the PR? I don't see it (was looking for `util.load_tx()` specifically and couldn't find the file)